spec/lib/services/sku-pack-service-spec.js

```diff
@@ -62,6 +72,116 @@ describe("SKU Pack Service", function() {
         fs.statSync.restore();
     });
 
+    it('should get the skus', function() {
+        waterline.skus.find.resolves({id: 'abc', sku: 'sku'});
+        skuService.getSkus({}).then(function(val){
+            expect(waterline.skus.find).to.have.been.called;
+            expect(val.id).equal('abc');
+            expect(val.sku).equal('sku');
+        });
+    });
+
+    describe('should get a sku by id', function(){
+        before(function(){
+            sinon.stub(skuService, 'getPackInfo');
+        });
+        after(function(){
+            skuService.getPackInfo.restore();
+        });
+        it('should get a sku by id', function() {
+            waterline.skus.needByIdentifier.resolves({id: 'abc', sku: 'sku'});
+            skuService.getPackInfo.resolves({
+                description: 'descrpt',
+                version: 'vers'
+            });
+            skuService.getSkusById('123').then(function(val){
```

Comment:
Promise need be returned in test case, otherwise  the mocha will not catch the assertion error.